app/routes/_layout+/assets.new.tsx

```diff
@@ -160,12 +169,13 @@ export default function NewAssetPage() {
   const title = useAtomValue(titleAtom);
   const [searchParams] = useSearchParams();
   const qrId = searchParams.get("qrId");
+  const { currency } = useLoaderData<typeof loader>();
 
   return (
     <>
       <Header title={title} />
       <div>
-        <AssetForm qrId={qrId} />
+        <AssetForm qrId={qrId} currency={currency} />
```

Comment:
Not sure why we are passing the currency here. It comes from the loader, just get it inside the asset form itself.